sklearn/inspection/partial_dependence.py

```diff
@@ -0,0 +1,664 @@
+    grid_resolution : int, optional (default=100)
+        The number of equally spaced points on the axes of the plots, for each
+        target feature.
+    percentiles : tuple of float, optional (default=(0.05, 0.95))
+        The lower and upper percentile used to create the extreme values
+        for the PDP axes. Must be in [0, 1].
+    method : str, optional (default='auto')
+        The method to use to calculate the partial dependence predictions:
+
+        - 'recursion' is only supported for objects inheriting from
+          `BaseGradientBoosting`, but is more efficient in terms of speed.
+          With this method, ``X`` is optional and is only used to build the
+          grid and the partial dependences are computed using the training
+          data. This method does not account for the ``init`` predicor of
+          the boosting process, which may lead to incorrect values (see
+          warning below. With this method, the target response of a
+          classifier is always the decision function, not the predicted
+          probabilities.
+
+        - 'brute' is supported for any estimator, but is more
+          computationally intensive.
+
+        - If 'auto', then 'recursion' will be used for
+          ``BaseGradientBoosting`` estimators with ``init=None``, and
+          'brute' for all other.
+
+        Unlike the 'brute' method, 'recursion' does not account for the
+        ``init`` predictor of the boosting process. In practice this still
+        produces the same plots, up to a constant offset in the target
+        response.
+    n_jobs : int, optional (default=None)
+        The number of CPUs to use to compute the partial dependences.
+        ``None`` means 1 unless in a :obj:`joblib.parallel_backend` context.
+        ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
+        for more details.
+    verbose : int, optional (default=0)
+        Verbose output during PD computations.
+    fig : Matplotlib figure object, optional (default=None)
+        A figure object onto which the plots will be drawn, after the figure
+        has been cleared.
+    line_kw : dict, optional
+        Dict with keywords passed to the ``matplotlib.pyplot.plot`` call.
+        For one-way partial dependence plots.
+    contour_kw : dict, optional
+        Dict with keywords passed to the ``matplotlib.pyplot.plot`` call.
+        For two-way partial dependence plots.
+    **fig_kw : dict, optional
+        Dict with keywords passed to the figure() call.
+        Note that all keywords not recognized above will be automatically
+        included here.
+
+    Examples
+    --------
+    >>> from sklearn.datasets import make_friedman1
+    >>> from sklearn.ensemble import GradientBoostingRegressor
+    >>> X, y = make_friedman1()
+    >>> clf = GradientBoostingRegressor(n_estimators=10).fit(X, y)
+    >>> fig, axs = plot_partial_dependence(clf, X, [0, (0, 1)]) #doctest: +SKIP
```

Comment:
Looks like we are not returning anything. Do we still want to use that Bunch idea from: https://github.com/scikit-learn/scikit-learn/issues/13448#issuecomment-483807326 which helps with implementing idea 2 of: https://github.com/scikit-learn/scikit-learn/issues/13448#issuecomment-479512520

Reply:
Indeed. This should be fixed in the narrative docs also.

Let's leave it returning None for now.